### DISCUSSION TITLE
Message when I run 'flet main.py'

### DISCUSSION THREAD

**Discussion by andrerotband** [Xynonners/fletbox#4] [General]
I would like to know if this is a warning or a problem:

(.venv) C:\src\flet\copahospeda>flet main.py

AlertDialog matched [2c 1u] possible subcontrol kwargs.
Banner matched [2c 0u] possible subcontrol kwargs.

**Comment by Xynonners** [comment]
It's just a warning due to the way certain things are done in the framework currently (try not to use the related controls in contextmanager a.k.a with statements, or else you will get unpredictable behavior). 

Using those controls in a with as long as you manually append however is fine, so e.g. (psuedocode)
```python
with YOUR_CONTROL_HERE as A:
    A.controls.append(YOUR_OTHER_CONTROL_HERE) #or wherever else it takes subcontrols
```

**Comment by andrerotband** [comment]
I am using it this way:

...
from views.despesas import DespesasView, DespesasViewDT
...

@fb.view("/despesas")
def pesquisa(page: ft.Page, builder: Builder) -> None:
    with builder.layout.Container(expand=True, margin=-10):
        with builder.layout.Column(
            horizontal_alignment=ft.CrossAxisAlignment.CENTER
        ) as col:
            col.controls.append(HeaderView(page, "Lançamento de Despesas"))
            col.controls.append(DespesasView())
            col.controls.append(DespesasViewDT())
